packages/SwingSet/src/kernel/state/vatKeeper.js

```diff
@@ -6,15 +10,34 @@ import { makeVatSlot, parseVatSlot } from '../../parseVatSlots';
 import { insistVatID } from '../id';
 import kdebug from '../kdebug';
 
-// makeVatKeeper is a pure function: all state is kept in the argument object
-
+/**
+ * Establish a vat's state.
+ *
+ * @param storage  The storage in which the persistent state will be kept
+ * @param deviceID  The device ID string of the device in question
+ *
+ * TODO: consider making this part of makeVatKeeper
+ */
 export function initializeVatState(storage, vatID) {
+  // TODO define constants for these various initial values (cf. kernelKeeper)
```

Comment:
I think @Chris-Hibbert 's recently-landed "vat stats" PR does exactly this, so when you rebase this to current trunk (on top of that change), and resolve the conflict created by this line, you might just remove the TODO note.

Reply:
Hasn't happened yet.  When it gets merged the TODO should be removed.